styles/styles.css

```diff
@@ -11,7 +11,11 @@
  */
 
 body {
-  background: var(--page-background);
+  background-color: var(--color-background);
+  background-size: 100% 34.5rem;
+  background-position: top;
+  background-repeat: no-repeat;
+  background-image: var(--page-background);
```

Comment:
this makes our gradients have a fixed height so they look correct on the article pages